### PR TITLE
Debug: trigger Read the Docs Addons event to test/debug locally

### LINKED ISSUE #1605: Mock the Read the Docs `CustomEvent` to render the selectors
In https://github.com/readthedocs/sphinx_rtd_theme/pull/1601 we added a version/language selector below the title using the Read the Docs `CustomEvent`. We want to figure it out a  way to be able to test this locally.

### PR DESCRIPTION
**Pull request by humitos**
Use a mocked JSON to trigger the event that Read the Docs Addons triggers when serving docs on production.

You can build the docs via:

```
sphinx-autobuild -E docs docs/_build/html
```

and then open http://127.0.0.1:8000 locally to see it in action.

Closes #1605

### CODE REVIEW

**Comment by agjohnson** [comment]
Are you not using `npm run dev`? There shouldn't be any manual commands necessary for development like that. The changes here look like they'll be usable either way, `npm run dev` doesn't do anything special during the Sphinx build.

File: docs/_static/debug.js [hunk 1]
```diff
@@ -4,4 +4,40 @@ $(function () {
   $("[data-toggle='rst-debug-badge']").on("click", function () {
     $("[data-toggle='rst-versions']").toggleClass("rst-badge");
   });
-})
+});
+
+
+const EVENT_READTHEDOCS_ADDONS_DATA_READY = "readthedocs-addons-data-ready";
+
+fetch("_static/addons.mocked.json", { method: "GET" })
```
Comment (agjohnson) [inline docs/_static/debug.js:12]: Seems this file is missing from this PR

**Review by agjohnson — APPROVED** [review]
Looks good besides the missing data :+1:

File: docs/_static/debug.js (hunk 1 above)
Comment (humitos) [reply docs/_static/debug.js:12]: Pushed!

**Comment by humitos** [comment]
> Are you not using `npm run dev`?

Interesting. I was not using this. I just tested it and it works great 💯

**Pull request merged**